code/modules/clothing/outfits/standard.dm

```diff
@@ -414,6 +414,16 @@
 	back = /obj/item/chrono_eraser
 	head = /obj/item/clothing/head/helmet/space/chronos
 	mask = /obj/item/clothing/mask/breath
+<<<<<<< HEAD
+=======
+	back = /obj/item/mod/control/pre_equipped/chrono
+
+/datum/outfit/chrono_agent/post_equip(mob/living/carbon/human/agent, visualsOnly)
+	. = ..()
+	var/obj/item/mod/control/mod = agent.back
+	var/obj/item/mod/module/eradication_lock/lock = locate(/obj/item/mod/module/eradication_lock) in mod.modules
+	lock.true_owner_ckey = agent.ckey
+>>>>>>> fd9a7f8a581 (MOD update: Modular Cores (#64042))
```

Comment:
merge this PR after https://github.com/Skyrat-SS13/Skyrat-tg/pull/10682 apparently

```suggestion
	back = /obj/item/mod/control/pre_equipped/chrono

/datum/outfit/chrono_agent/post_equip(mob/living/carbon/human/agent, visualsOnly)
	. = ..()
	var/obj/item/mod/control/mod = agent.back
	var/obj/item/mod/module/eradication_lock/lock = locate(/obj/item/mod/module/eradication_lock) in mod.modules
	lock.true_owner_ckey = agent.ckey
```